src/features/events/components/EventTypeAutocomplete.tsx

```diff
@@ -33,6 +33,8 @@ const useStyles = makeStyles<Theme, StyleProps>((theme) => ({
 }));
 
 type EventTypeAutocompleteProps = {
+  inEventCreationModal?: boolean;
```

Comment:
I'm not sure about this property, since it's so very specific. Maybe it could be replaced with properties for the actual configurable options, e.g. `sx` (for the styling bits), `size`, `label` and `shrink`.

Reply:
Looking into the merge conflict, I'm realizing this component `EventTypeAutocomplete` is probably not a good basis for whatever we want to do in this modal at all. I would create a separate component for this modal.